ui/src/containers/streams/EditStream.js

```diff
@@ -219,8 +219,9 @@ class EditStream extends Component {
                   Name<span className="text-danger ms-1">*</span>
                 </h5>
                 <span className="text-muted fs-7 me-2">
-                  Name of the Apache Kafka topic. If the topic does not yet
-                  exist on the broker, we will automatically create it.
+                  Name of the Apache Kafka topic. You cannot update the field{" "}
```

Comment:
Why is there a space inserted here? Would it make sens to either insert into `<i>`? Inserting a React construct for having a space in HTML feels unnecessary.

Reply:
In this case, I can blame [prettier](https://prettier.io/) which I ran before committing the changes. 😆

Reply:
Oh wow! :D